agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/configuration/Configuration.java

```diff
@@ -96,7 +96,8 @@
 
         public String name;
         public String objectName;
-        public String attribute;
+        public String attribute = "";
+        public List<String> attributes = new ArrayList<>();
```

Comment:
can you add comment here explaining why there is both attribute and attributes?

Reply:
Done